packages/ui-billboard/src/Billboard/__tests__/Billboard.test.tsx

```diff
@@ -43,10 +43,7 @@ describe('<Billboard />', async () => {
   })
 
   describe('with generated examples', async () => {
-    generateA11yTests(
-      Billboard,
-      BillboardExamples as StoryConfig<BillboardProps>
-    )
+    generateA11yTests(Billboard, BillboardExamples)
```

Comment:
Just remove the unused imports here (BillboardProps and StoryConfig) and it should be fine :) Good job!